<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDPClient</class>
 <widget class="QWidget" name="UDPClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDPClient</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_SendMessage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>471</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Port">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Send">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>581</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>321</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.137.177</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>IP Address:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
